misk/web/tabs/loader/src/containers/LoaderContainer.tsx

```diff
@@ -0,0 +1,79 @@
+import { IconName, IconNames } from "@blueprintjs/icons"
+import * as React from "react"
+import { connect } from "react-redux"
+import { Route, Switch } from "react-router"
+import styled from "styled-components"
+import { IAppState } from ".."
+import { dispatchAdminTabs } from "../actions"
+import { NoMatchComponent, ScriptComponent } from "../components"
+
+interface ITabProps {
+  adminTabs: IAdminTabs
+  loading: boolean
+  error: any
+  getTabs: any
+}
+
+export interface ILoaderState {
+  adminTabs: IAdminTabs
+}
+
+export interface IAdminTabs {
+  [key:string]: IAdminTab
+  toJS: any
+}
+
+export interface IAdminTab {
+  icon: IconName
+  name: string
+  slug: string
+  url_path_prefix: string
+}
+
+const Container = styled.div``
```

Comment:
Accidental template string instead of semicolon? This kind of thing would be caught with [tslint](https://github.com/palantir/tslint) and/or [prettier](https://www.github.com/prettier/prettier).

Reply:
Looks like it might be intentional, but a little confusing. I still like the idea of using tslint & prettier.

Reply:
tslint is used with extensive rules though I agree that the empty styles is a confusing syntax so I'll be only using styled divs when the div has styles to be declared.